packages/tpetra/core/src/Tpetra_Vector_decl.hpp

```diff
@@ -255,16 +255,6 @@ class Vector : public MultiVector<Scalar, LocalOrdinal, GlobalOrdinal, Node>
   ///   assignment, and move assignment.
   virtual ~Vector () = default;
 
-  //@}
-  //! \name Clone method
-  //@{
-
-  /// \brief Return a deep copy of <tt>*this</tt> with a different
-  ///   Node type (and therefore a different Device type).
-  /// \tparam Node2 The returned Vector's Node type.
-  ///
-  /// \param node2 [in] The returned Vector's Kokkos Node instance.
-//@}
   //! @name Post-construction modification routines
```

Comment:
There's probably a missing `//@}` above this.  I saw another one of these above.